src/test_driver/openiotsdk/integration-tests/all-clusters-app/test_app.py

```diff
@@ -0,0 +1,147 @@
+    assert controller != None
+    devCtrl = controller
+
+    ret = device.wait_for_output("Open IoT SDK all-clusters-app example application start")
+    assert ret != None and len(ret) > 0
+
+    setupPayload = get_setup_payload(device)
+    assert setupPayload != None
+
+    ret = device.wait_for_output("Current software version")
+    assert ret != None and len(ret) > 1
+
+    version_app = ret[-1].split()[-2:]
+
+    commissionable_device = discover_device(devCtrl, setupPayload)
+    assert commissionable_device != None
+
+    nodeId = connect_device(devCtrl, setupPayload, commissionable_device)
+    assert nodeId != None
+
+    ret = device.wait_for_output("Commissioning completed successfully")
+    assert ret != None and len(ret) > 0
+
+    acs = AccessControl.Structs()
+    ace = AccessControl.Enums()
+
+    #  AccessControl cluster test - write entires
+    # entry0=acs.AccessControlEntryStruct(privilege=ace.AccessControlEntryPrivilegeEnum.kAdminister, authMode=ace.AccessControlEntryAuthModeEnum.kCase, subjects=NullValue, targets=[acs.Target(cluster=NullValue, endpoint=0, deviceType=NullValue), acs.Target(cluster=1, endpoint=NullValue, deviceType=NullValue), acs.Target(cluster=2, endpoint=3, deviceType=NullValue)], fabricIndex=1)
+    # entry1=acs.AccessControlEntryStruct(privilege=ace.AccessControlEntryPrivilegeEnum.kView, authMode=ace.AccessControlEntryAuthModeEnum.kCase, subjects=[4, 5, 6, 7], targets=[acs.Target(cluster=NullValue, endpoint=8, deviceType=NullValue), acs.Target(cluster=9, endpoint=NullValue, deviceType=NullValue), acs.Target(cluster=10, endpoint=11, deviceType=NullValue)], fabricIndex=1)
+    # entry2=acs.AccessControlEntryStruct(privilege=ace.AccessControlEntryPrivilegeEnum.kOperate, authMode=ace.AccessControlEntryAuthModeEnum.kGroup, subjects=[12, 13, 14, 15], targets=[acs.Target(cluster=NullValue, endpoint=16, deviceType=NullValue), acs.Target(cluster=17, endpoint=NullValue, deviceType=NullValue), acs.Target(cluster=18, endpoint=19, deviceType=NullValue)], fabricIndex=1)
+    # entry3=acs.AccessControlEntryStruct(privilege=ace.AccessControlEntryPrivilegeEnum.kOperate, authMode=ace.AccessControlEntryAuthModeEnum.kCase, subjects=[20, 21, 22, 23], targets=[acs.Target(cluster=NullValue, endpoint=24, deviceType=NullValue), acs.Target(cluster=25, endpoint=NullValue, deviceType=NullValue), acs.Target(cluster=26, endpoint=27, deviceType=NullValue)], fabricIndex=1)
+    # err, res = write_zcl_attribute(devCtrl, "AccessControl", "Acl", nodeId,  ALL_CLUSTERS_CTRL_TEST_ENDPOINT_ID,
+    #                                [entry0, entry1, entry2, entry3])
+    # assert err == 0
+    # assert res[0].Status == chip.interaction_model.Status.Success
+
+    # err, res = read_zcl_attribute(devCtrl, "AccessControl", "Acl", nodeId, ALL_CLUSTERS_CTRL_TEST_ENDPOINT_ID)
+    # assert err == 0
+    # assert res.value[0] == entry0
+    # assert res.value[1] == entry1
+    # assert res.value[2] == entry2
+    # assert res.value[3] == entry3
```

Comment:
Do we need that commented out code ?

Reply:
Of course not. It was only for testing. Restored